docs/podman-container-checkpoint.1.md

```diff
@@ -10,7 +10,7 @@ podman\-container\-checkpoint - Checkpoints one or more running containers
 Checkpoints all the processes in one or more containers. You may use container IDs or names as input.
 
 ## OPTIONS
-**--keep**, **-k**
+**--keep, -k**
```

Comment:
Thought we'd a recent request to NOT bold the comma in situations like this?  Is that against the standard?